app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -151,16 +151,21 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
     }
 
     fun completeOrder(charge: Charge) {
-        compositeDisposable.add(orderService.chargeOrder(orderIdentifier, charge)
+        compositeDisposable.add(orderService.chargeOrder(orderIdentifier.toString(), charge)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .doOnSubscribe {
                     progress.value = true
                 }.doFinally {
                     progress.value = false
                 }.subscribe({
-                    message.value = "Payment completed successfully!"
-                    Timber.d("Success placing order!")
+                    if (it.status != null && it.status) {
+                        message.value = it.message
+                        Timber.d("Successfully  charged for the order!")
+                    } else {
+                        message.value = it.message
```

Comment:
This call is in both blocks

What's the use of if statement

Reply:
Updated the PR but I am still using the if statements to see if the status is true/ false to see if the payment is successful  or not. Please review.